spring/boot2-autoconfigure/src/test/resources/config/application-deprecatedIpTest.yml

```
@@ -0,0 +1,8 @@
+armeria:
+  ports:
+    - ip: 127.0.0.1
```

optional; I'm unsure if this would also work in windows, but I think it would be cool if we can check that non-ip addresses are also accepted

```suggestion
    - address: localhost
```